.github/workflows/workflow-build-push-container-azure.md

```diff
@@ -3,6 +3,6 @@
    - **Purpose:** Build and push a docker container to Azure Container Registry (ACR).
    - **Usage:** Call this workflow and provide the `container-name` and `tag` as inputs.
    - **Required Secrets:**
-     - `ACR_REGISTRY_NAME`: The Azure container registry name.
+     - `ACR_REGISTRY_LINK`: The Azure container registry link.
```

Comment:
what does a ACR_REGISTRY_LINK look like?

Reply:
it's a link they give you
![Screenshot_63](https://github.com/ai-cfia/github-workflows/assets/9827730/fae352fe-b9dc-48a3-914b-82848aeab49c)
we can keep the name but we need to change the workflow to
{{ ACR_REGISTRY_NAM }}'.azurecr.io'

Reply:
just saw the other comment should be URL I can change that but just needs to know which option is prefered

Reply:
ACR means Azure Container Registry. So we have registry twice in the name.

Based on your example, it's not an URL then (URL specificies the protocol), this is a DNS name.

So more accurate ACR_SERVER or AZURE_CONTAINER_REGISTRY_SERVER and put the full domain name `aciacfiaacr.azurecr.io` for it (don't take shortcut as prefix as they might very well decide to add additional domains in the future).

As well, provide examples in the documentation (also for inputs)

Reply:
I wasn't the one who created it but I can change it. Looking at the GCP one we have 
registry: ${{ secrets.GCP_CLOUDRUN_REGION }}-docker.pkg.dev
i`m going to restructure the workflows to have only one workflow to build

Reply:
ACR_REGISTRY is still redundant as the R in ACR stands for Registry.